### ISSUE TITLE
More thumbnail work - images already in project.

### ISSUE THREAD

**Issue by ldecicco-USGS** [USGS-VIZLAB/vizlab#370]
If we have an image already in a project, we need to load it a second time to use it as a thumbnail (see water-use-15).

**Comment by jread-usgs** [comment]
The thumbnail would be the same resolution as the image in the viz?

**Comment by ldecicco-USGS** [comment]
Yeah, facebook and twitter actually allow for pretty big images. It's mainly our "landing" page that needs a very specific sized one.
